stake-pool/cli/src/main.rs

```diff
@@ -1769,9 +2097,17 @@ fn main() {
                     .required(true)
                     .help("Stake pool address."),
             )
+            .arg(Arg::with_name("fee_type")
+                .index(2)
+                .value_name("FEE_TYPE")
+                .possible_values(&["pool","deposit", "withdrawal"]) // PreferredValidatorType enum
```

Comment:
There's three different names for the original fee at this point: management, pool, and epoch.  `pool` is probably too vague, so I'm happy with either `management` or `epoch`, with a slight preference toward `epoch`.  Either way, let's keep it consistent

Reply:
ok, epoch it is.